### PR TITLE
remove addRoutePattern calls

### PR DESCRIPTION
**Pull request by aloupfor**

### CODE REVIEW

**Comment by aloupfor** [comment]
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1216

**Comment by aloupfor** [comment]
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1216

**Comment by ghost** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1:

File: MediaAdminBundle/Resources/public/coffee/LoadBaseConfigurationRoute.coffee [hunk 1]
```diff
@@ -1,8 +1,4 @@
 ((router) ->
-  router.addRoutePattern('listFolder', 'folder/:folderId/list');
-  router.addRoutePattern('addMedia', 'folder/:folderId/add/media');
-  router.addRoutePattern('mediaEdit', 'folder/:folderId/list/media/:mediaId/edit');
```
Comment (ngilain) [inline MediaAdminBundle/Resources/public/coffee/LoadBaseConfigurationRoute.coffee:4]: What's going on for the routes addMedia and mediaEdit ?

Comment (ngilain) [inline MediaAdminBundle/Resources/public/coffee/LoadBaseConfigurationRoute.coffee:4]: Ok seen in the hidden code

**Comment by ngilain** [comment]
:+1:

**Comment by aloupfor** [comment]
https://github.com/open-orchestra/open-orchestra-docs/pull/128

**Pull request merged**